service/history/hsm/registry.go

```diff
@@ -171,21 +167,6 @@ func (r *Registry) ExecuteActiveImmediateTask(
 	return r.execute(ctx, executor, env, ref, task)
 }
 
-// ExecuteStandbyImmediateTask gets an [ImmediateExecutor] from the registry and invokes it.
-// Returns [ErrNotRegistered] if an executor is not registered for the given task's type.
-func (r *Registry) ExecuteStandbyImmediateTask(
```

Comment:
Please remove `ExecuteStandbyTimerTask` and fix timer task registration to only accept the active executor.

Reply:
Gonna do in a separate PR.